src/Module.h

```diff
@@ -2,6 +2,9 @@
 #define _RADIOLIB_MODULE_H
 
 #include "TypeDef.h"
+#include <functional>
+#include <FunctionalInterrupt.h>
```

Comment:
I'm not sure why this is needed but the signature of `attachInterrupt` without this is `void attachInterrupt(uint8_t, void (*)(), int)`. Is it supposed to be like that or is it an error of mine of not having the appropriate compiling includedirs?

Reply:
`<functional>` is not supported on AVR Arduino - see this failing run: https://github.com/jgromes/RadioLib/pull/290/checks?check_run_id=2428779662

Looking into AVR Arduino `attachInterrupt` implementation, it is hard-coded to only accept ISRs without `void*` arguments. So this will never work on AVR Arduinos like Uno, which are unfortuantely still the most common ones.